src/AaveV2EthBUSDIR_20230602/DeployBUSDOffBoard.s.sol

```diff
@@ -14,11 +14,17 @@ contract BUSDPayloadProposal is EthereumScript {
   function run() external broadcast {
     GovHelpers.Payload[] memory payloads = new GovHelpers.Payload[](1);
     payloads[0] = GovHelpers.buildMainnet(
-      address(0) // TODO: Replace by actual payload
+      address(new AaveV2EthBUSDIR_20230602()) // TODO: Replace by actual payload
     );
     GovHelpers.createProposal(
       payloads,
-      0 // TODO: replace by actual Hash
+      GovHelpers.ipfsHashFile(
```

Comment:
instead of putting the ipfs hash directly you can just put the relative path(relative to the repo root) which upon execution wil log sth like:
> == Logs ==
  Info: This preview will only work when the file has been uploaded to ipfs
  Preview: https://app.aave.com/governance/ipfs-preview/?ipfsHash=0xfd26bc5c71ebf951632d2f7b156076175ba85cdb4eb6dc090f949a3b03c771f0

- the preview will be reachable once the pr is merged to main
- if you want to see the preview earlier you can set the flag to `true` and add `PINATA_KEY` & `PINATA_SECRET` to your `.env`